packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -694,3 +694,12 @@ module.exports.newsletters = isA
   )
   .default([])
   .optional();
+
+module.exports.thirdPartyProvider = isA
+  .string()
+  .max(256)
+  .allow(['google', 'apple'])
```

Comment:
Just making sure that allowing 'apple' was intentional, since I don't see it being handled by explicitly by the code.